packages/forms-web-app/__tests__/unit/views/lpa/confirm-remove-user.njk.test.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-env browser */
+const { VIEW } = require('../../../../src/lib/views');
+const nunjucksTestRenderer = require('../nunjucks-render-helper');
+
+describe('views/lpa/confirm-remove-user', () => {
+	describe('when a user navigates to the confirm-remove-user page', () => {
+		it('should render the page title as expected', () => {
+			document.body.innerHTML = nunjucksTestRenderer.render(
+				`${VIEW.LPA_DASHBOARD.CONFIRM_REMOVE_USER}.njk`
+			);
+			expect(document.querySelector('title').textContent.trim()).toEqual(
+				`Confirm you want to remove this user - Manage appeals - GOV.UK`
+			);
+		});
+		it('should render the page heading as expected', () => {
+			document.body.innerHTML = nunjucksTestRenderer.render(
+				`${VIEW.LPA_DASHBOARD.CONFIRM_REMOVE_USER}.njk`
+			);
+
+			console.log(document.body.innerHTML);
+
+			expect(document.querySelector('#main-content h1').textContent.trim()).toEqual(
+				'Confirm you want to remove this user'
+			);
+		});
+
+		it('should render the email address as expected', () => {
```

Comment:
I'm also not sure on these tests, are we just testing the nun jucks rendering engine and the gov uk templates here in places? I think the main value is in testing the the nun jucks renderer has been called with the expected values that we pass in. Once we integrate the cypress tests we might also be duplicating effort here. Again not a blocker just food for thought.

Reply:
We previously were doing snapshots of the views, which I did think was pointless as you just updated the snapshots mindlessly
Charlie started adding tests like this and felt it was a little more useful.
Happy to remove going forward.
This was some of the things I wanted to raise so glad you largely agree